source/org/zfin/mapping/MappingService.java

```diff
@@ -144,6 +149,17 @@ private static List<GenomeLocation> sortAndFilterGenomeBrowserLocations(List<? e
         }
         return finalGenomeList;
     }
+
+    public static List<MarkerLocation> getMarkerLocation(String zdbID) {
+        Session session = HibernateUtil.currentSession();
```

Comment:
I noticed this is still using the old version of org.hibernate.Criteria.  I assume that you are saving that for later as we make our way through the various repositories?